lib/install/validate/config.go

```diff
@@ -38,7 +39,7 @@ func (v *Validator) CheckFirewall(ctx context.Context) {
 	var err error
 
 	rule := types.HostFirewallRule{
-		Port:      8080, // serialOverLANPort
+		Port:      exec.SerialOverLANPort, // serialOverLANPort
```

Comment:
we really should try to avoid pulling the entire portlayer into the installer just for constants.
Can we split this into a seperate portlayer/exec/consts package if this is necessary.